src/base/orgs/View/Anchor.svelte

```diff
@@ -0,0 +1,211 @@
+<script lang="ts">
+  import { ethers } from "ethers";
+  import type { Safe } from "@app/utils";
+  import type { PendingProject } from "@app/project";
+  import type { Session } from "@app/session";
+  import type { Config } from "@app/config";
+  import * as utils from "@app/utils";
+  import Modal from "@app/Modal.svelte";
+  import Avatar from "@app/Avatar.svelte";
+
+  export let safe: Safe;
+  export let project: PendingProject;
+  export let session: Session;
+  export let config: Config;
+
+  enum State {
+    Idle,
+    Confirm,
+    Signing,
+    Submitting,
+    Success,
+    Execute,
+    Failed,
+  }
+
+  enum Action {
+    Sign,
+    Execute,
+  }
+
+  let state = State.Idle;
+  let action: null | Action = null;
+  const isSigned = project.confirmations.includes(
+    ethers.utils.getAddress(session.address)
+  );
+  const close = () => {
+    action = null;
+    state = State.Idle;
+  };
+  const pending = safe.threshold - project.confirmations.length;
```

Comment:
I would check the confirmations is `<` than threshold first. It's possible for there to be more confirmations than the threshold.

Reply:
Nevermind, you do a `<=` check lower.